pkgs/shells/zsh/default.nix

```diff
@@ -55,9 +55,11 @@ stdenv.mkDerivation {
   checkFlags = map (T: "TESTNUM=${T}") (lib.stringToCharacters "ABCDEVW");
 
   # XXX: think/discuss about this, also with respect to nixos vs nix-on-X
-  postInstall = ''
+  postInstall = lib.optionalString stdenv.isDarwin ''
+    make install.bin install.modules install.fns
```

Comment:
Shouldn't this be already a part of the installPhase?

Reply:
Yes - I was so busy trying to fix the build I didn't even think about it. See #154403.